src/alignment/mod.rs

```diff
@@ -229,6 +231,8 @@ mod tests {
             score: 5,
             xstart: 3,
             ystart: 0,
+            xend: 10,
+            yend: 10,
```

Comment:
That's fine. Lets keep all of them. We just need to make sure that they are properly documented. And xlen can probably be private.